Source/Scene/Points3DTileContentProvider.js

```diff
@@ -64,66 +68,76 @@ define([
 
         this.state = Cesium3DTileContentState.LOADING;
 
-        function failRequest(error) {
+        loadArrayBuffer(this._url).then(function(arrayBuffer) {
+            that.init(arrayBuffer);
+        }).otherwise(function(error) {
             that.state = Cesium3DTileContentState.FAILED;
             that.readyPromise.reject(error);
+        });
+    };
+
+    Points3DTileContentProvider.prototype.init = function(arrayBuffer, byteOffset) {
+        byteOffset = defaultValue(byteOffset, 0);
+
+        var magic = getMagic(arrayBuffer, byteOffset);
```

Comment:
Can you check if all the other tile formats, including the new composite, and verifying the magic number like this?